### PR TITLE
Update 'ps' (Pashto) locale

### LINKED ISSUE #3147: Timestamps for Pashto locale
**Is your feature request related to a problem? Please describe.**
We need to ensure timestamps support Pashto, this includes correct translations and numerals.

When viewing an article in Pashto, the timestamp should:
- Use [Eastern Arabic numerals](https://en.wikipedia.org/wiki/Eastern_Arabic_numerals)
- Display both relative and absolute timestamps in Pashto
- Display updated timestamps correctly in Pashto
- Include both Gregorian and Jalaali calendar dates (already enabled in https://github.com/bbc/simorgh/pull/3552)
- Be rendered right-to-left
- Have onscreen and offscreen translations in Pashto

**Other links**
Storybook for examples of relative timestamps, e.g. https://bbc.github.io/simorgh/?path=/story/containers-article-article-timestamp--lastpublished-firstpublished-and-firstpublished-10-hours-ago

Pashto article fixture data: https://github.com/bbc/simorgh/tree/latest/data/pashto/articles

**Describe the solution you'd like**
Update the [`ps` locale in psammead-locales](https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-locales/moment/ps.js) to support the above requirements.

**Note** the locale does not need to include all Moment locale features, only enough to support our timestamp formats. 

**Describe alternatives you've considered**
None.

**Testing notes**
- Requires unit test coverage for Pashto locale

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by simonsinclair**
Resolves https://github.com/bbc/simorgh/issues/3147

**Overall change:** Update the Pashto locale to support the editorial whitelisted relative time permutations and test them.

**Code changes:**

- Add Pashto relative time translations for:
  - a few seconds ago
  - a minute ago
  - [n] minutes ago
  - an hour ago
  - [n] hours ago
- Add "from" test to validate the above.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by RayNjeri — APPROVED** [review]

File: packages/utilities/psammead-locales/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.11.0 | [PR#2178](https://github.com/bbc/psammead/pull/2178)  Update `ps` (Pashto) locale |
```
Comment (jamesdonoh) [inline packages/utilities/psammead-locales/CHANGELOG.md]: Please could you briefly say what we've changed, e.g. `...to add relative timestamp support`?

File: packages/utilities/psammead-locales/moment/ps.js [hunk 2]
```diff
@@ -24,9 +24,18 @@ var pashtoGregorianMonths = 'جنوري_فبروري_مارچ_اپریل_می_ج
...
+
+  relativeTime: {
+    past: '%s مخکې', // %s 'ago'
+    s: 'یو څو ثانیې', // 'a few seconds'
+    m: 'یوه دقیقه', // 'a minute'
+    mm: '%d دقیقې', // %d 'minutes'
+    h: 'یو ساعت', // 'an hour'
+    hh: '%d ساعتونه', // %d 'hours'
+  },
+
```
Comment (jamesdonoh) [inline packages/utilities/psammead-locales/moment/ps.js]: Think we should delete the blank line here and L29 for consistency with the other locales.

File: packages/utilities/psammead-locales/moment/ps.test.js [hunk 3]
```diff
@@ -23,3 +23,17 @@ test('format', function () {
...
+    assert.equal(start.from(moment([2007, 1, 28]).add({s: 44}), true), 'یو څو ثانیې', '44 seconds = a few seconds');
+    assert.equal(start.from(moment([2007, 1, 28]).add({s: 45}), true), 'یوه دقیقه',   '45 seconds = a minute');
+    assert.equal(start.from(moment([2007, 1, 28]).add({s: 89}), true), 'یوه دقیقه',   '89 seconds = a minute');
+    assert.equal(start.from(moment([2007, 1, 28]).add({s: 90}), true), '۲ دقیقې',     '90 seconds = 2 minutes');
+    assert.equal(start.from(moment([2007, 1, 28]).add({m: 44}), true), '۴۴ دقیقې',    '44 minutes = 44 minutes');
+    assert.equal(start.from(moment([2007, 1, 28]).add({m: 45}), true), 'یو ساعت',     '45 minutes = an hour');
+    assert.equal(start.from(moment([2007, 1, 28]).add({m: 89}), true), 'یو ساعت',     '89 minutes = an hour');
+    assert.equal(start.from(moment([2007, 1, 28]).add({m: 90}), true), '۲ ساعتونه',   '90 minutes = 2 hours');
+    assert.equal(start.from(moment([2007, 1, 28]).add({h: 5}), true),  '۵ ساعتونه',   '5 hours = 5 hours');
+    assert.equal(start.from(moment([2007, 1, 28]).add({h: 21}), true), '۲۱ ساعتونه', '21 hours = 21 hours');
```
Comment (jamesdonoh) [inline packages/utilities/psammead-locales/moment/ps.test.js:38]: Could we either align the spacing before the final argument or make it all just 1sp for readability?

**Review by jamesdonoh — CHANGES_REQUESTED** [review]
Looking good, just a few small quibbles

**Review by sareh — APPROVED** [review]
Great work. 😄 The pashto locale itself reads correctly. 👍

**Review by jamesdonoh — APPROVED** [review]

**Pull request merged**